packages/collector/test/tracing/native_esm/test.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../../core/test/config');
+const { retry, verifyEntrySpan } = require('../../../../core/test/test_util');
+const ProcessControls = require('../../test_util/ProcessControls');
+const globalAgent = require('../../globalAgent');
+const constants = require('@instana/core').tracing.constants;
+const path = require('path');
+const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
+
+mochaSuiteFn('tracing/native-esm modules', function () {
+  this.timeout(config.getTestTimeout());
+
+  globalAgent.setUpCleanUpHooks();
+  const agentControls = globalAgent.instance;
+
+  let controls;
+
+  before(async () => {
+    controls = new ProcessControls({
+      dirname: __dirname,
+      useGlobalAgent: true,
+      env: {
+        INSTANA_CUSTOM_INSTRUMENTATIONS: [path.resolve(__dirname, './fakeInstrumentation/squareCalc')]
```

Comment:
Why calling it "fake"? This is a custom instrumentation.

Reply:
renamed to customInstrumentation